doc/source/index.rst

```diff
@@ -17,7 +17,7 @@ It originates from `a fork of retrying
 The simplest use case is retrying a flaky function whenever an `Exception`
 occurs until a value is returned.
 
-.. testcode::
+.. code-block:: python
```

Comment:
sounds to me that it won't use doctest anymore if we change that, no?

Reply:
Yep. For doctests specific tags present, but I don't remember for usage possibility for readme.

Reply:
So that's not gonna fly. Best thing is to just replace the README.rst by a link to the doc at this stage.

Reply:
Ok, closing PR